autoload/SpaceVim/layers/lang/go.vim

```diff
@@ -0,0 +1,12 @@
+function! SpaceVim#layers#lang#go#plugins() abort
+    if has('nvim')
+        return ['zchee/deoplete-go', {'on_ft' : 'go', 'build': 'make'}]
+    else
+        return ['fatih/vim-go', { 'on_ft' : 'go', 'loadconf_before' : 1}]
```

Comment:
@phacops I have enable vim-go for neovim, but in neovim, deoplete is the default completion tools , so I use deoplete-go source, and ignore omni func from vim-go.
@zchee I want to use nvim-go for neovim though, if it is stabe please let me know, thanks for your work on nvim-go.